### PR TITLE
commiting perl-threaded fix

### PR DESCRIPTION
**Pull request by jerowe**
* [x ] I have read the guidelines above.
* [x ] This PR updates an existing recipe.

https://github.com/bioconda/bioconda-recipes/pull/1443/files/6d82622a5a5bcc2761f6f7071f2ab54fc53f1533#r61863884

I changed perl to perl-threaded.

### CODE REVIEW

**Comment by bgruening** [comment]
You need to increase the build number.
Thanks!

**Comment by jerowe** [comment]
Thanks @bgruening . I updated build number.

File: recipes/perl-biox-workflow/meta.yaml [hunk 1]
```diff
@@ -10,20 +10,20 @@ source:
    # List any patch files here
    # - fix.patch
 
-# build:
+ build:
   # If this is a new build for the same version, increment the build
   # number. If you do not include this key, it defaults to 0.
-  # number: 1
+   number: 1
```
Comment (bgruening) [inline recipes/perl-biox-workflow/meta.yaml]: I guess here is an indentation error.

**Comment by bgruening** [comment]
Great! Thanks a lot!

**Pull request merged**